### PR TITLE
UI Kit | FE | TextInput add new isCondensed prop

### PR DESCRIPTION
**Pull request by jmcreasman**
Adding a new boolean isCondensed prop that when set to true condenses the input height and font size.

isCondensed = false
Screenshot 2024-04-11 at 11 22 39 AM

isCondensed = true
Screenshot 2024-04-11 at 11 22 46 AM

### CODE REVIEW

**Comment by jmcreasman** [comment]
@CarlosCortizasCT does SHIELD handle updating the Percy tests or is that something I'm expected to do? Thanks!

**Review by FilPob — APPROVED** [review]

**Comment by CarlosCortizasCT** [comment]
> @CarlosCortizasCT does SHIELD handle updating the Percy tests or is that something I'm expected to do? Thanks!

Hi @jmcreasman 

I'm not sure what you mean, sorry.

You've already updated the visual regression tests in your PR and the differences in Percy has already been approved by Filip.

If you mean approving the differences in Percy, we expect @FilPob to do that once he checks everything looks as expected.

**Review by kark — APPROVED** [review]
Thanks 🙌
Aside from what Carlos mentioned, it looks good to me. I'll go ahead and approve.

File: packages/components/inputs/input-utils/src/styles.ts [hunk 1]
```diff
@@ -78,9 +79,15 @@ const getInputStyles = (props: TInputProps) => {
     display: flex;
     flex: 1;
     font-family: inherit;
-    font-size: ${designTokens.fontSize30};
-    height: ${designTokens.heightForInput};
-    min-height: ${designTokens.heightForInput};
+    font-size: ${props.isCondensed
+      ? `${designTokens.fontSize20}`
+      : `${designTokens.fontSize30}`};
```
Comment (ddouglasz) [inline packages/components/inputs/input-utils/src/styles.ts:84]: Seems like we are making these changes in multiple components, maybe we can have a token for the condensed state font size.
```suggestion
    font-size: ${props.isCondensed
      ? `${designTokens.fontSizeForInputWhenCondensed}`
      : `${designTokens.fontSize30}`};
```

**Review by ddouglasz — COMMENTED** [review]
Looks good to me 💯
Just a minor observation to consider.

**Review by ddouglasz — APPROVED** [review]

File: packages/components/inputs/text-input/src/text-input.tsx [hunk 2]
```diff
@@ -48,6 +48,10 @@ export type TTextInputProps = {
    * Focus the input on initial render
    */
   isAutofocussed?: boolean;
+  /**
+   * Whether the options appear in a condensed layout
```
[the remark this answers is not in the document]
Comment (jmcreasman) [reply packages/components/inputs/text-input/src/text-input.tsx:52]: Sure thing!

File: .changeset/famous-shrimps-change.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/input-utils': minor
+'@commercetools-uikit/text-input': minor
+---
+
+"feat(text-input): add new isCondensed prop"
```
[the remark this answers is not in the document]
Comment (jmcreasman) [reply .changeset/famous-shrimps-change.md:6]: Done!

File: packages/components/inputs/text-input/src/text-input.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply packages/components/inputs/text-input/src/text-input.tsx]: You need to generate the README again so it takes the updated comment.

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks! 👍

**Pull request merged**